server/schedulers/hot_region.go

```diff
@@ -473,6 +481,12 @@ func (bs *balanceSolver) solve() []*operator.Operator {
 	}
 	bs.cur = &solution{}
 	tryUpdateBestSolution := func() {
+		if bs.cur.progressiveRank == -1 && bs.isUniformFirstPriority(bs.cur.srcStore) {
```

Comment:
`bs.isUniformFirstPriority(bs.cur.srcStore)` can be computed only once after `for _, srcStore := range bs.filterSrcStores()`?